dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/sip/netty/SipTcpInboundConnLink.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2008, 2009 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.sip.stack.transport.sip.netty;
+
+import io.netty.channel.Channel;
+
+/**
+ * represents an inbound tcp connection
+ * 
+ * @author ran
+ */
+public class SipTcpInboundConnLink extends SipInboundConnLink {
+	/**
+	 * constructor
+	 * 
+	 * @param sipInboundChannel channel that created this connection
+	 * @param channel a Netty channel associated with this inbound SIP channel
+	 */
+	public SipTcpInboundConnLink(SipInboundChannel sipInboundChannel, Channel channel) {
+		super(sipInboundChannel, channel);
+	}
+
+
+	// ----------------------------
+	// SIPConnection implementation
+	// ----------------------------
+
+	/**
+	 * @see com.ibm.ws.sip.stack.transaction.transport.connections.SIPConnection#getTransport()
+	 */
+	public String getTransport() {
+		return "tcp";
```

Comment:
Constant value for 'tcp' and 'udp'. We should be consistent throughout the component since they are widely used.

Reply:
added existing constant